src/lib/storage/session.js

```diff
@@ -235,6 +238,31 @@ SessionStorage.prototype = {
   },
 
   /**
+   * Get the selected tab index for a group.
+   *
+   * @param {ChromeWindow} chromeWindow
+   * @param {Number} groupID - the groupID
+   */
+  getGroupSelectedIndex: function(chromeWindow, groupID) {
+    let groupsData = this._getGroupsData(chromeWindow);
+    let currentGroup = groupsData[groupID];
```

Comment:
`groupsData` could return an empty object for newly created sessions, so `groupsData[groupID]` is undefined.